frontend/src/components/bucket/BucketSidebar.vue

```diff
@@ -10,13 +16,41 @@ type Props = {
 
 const props = withDefaults(defineProps<Props>(), {});
 
+//Store
+const permissionStore = usePermissionStore();
+const userStore = useUserStore();
+
+const { getBucketPermissions } = storeToRefs(permissionStore);
+const { userSearch } = storeToRefs(userStore);
+
+// State
+const managedBy: Ref<string | undefined> = ref();
+
 // Emits
 const emit = defineEmits(['close-sidebar-info']);
```

Comment:
Please move the Emits to after the Props. This is not outlined anywhere, but a pattern I am moving to for consistency is Props/Emits/Store/State/Actions.